src/server/worldserver/worldserver.conf.dist

```diff
@@ -796,7 +797,7 @@ CharactersPerRealm = 12
 #
 #    HeroicCharactersPerRealm
 #        Description: Limit number of heroic class characters per account on this realm.
-#        Range:       1-10
+#        Range:       1-12
 #        Default:     1
```

Comment:
I don't know WTF am I writing about... I just re-checked the commit and saw that there IS already `DemonHuntersPerRealm` present.